upgrade-2.x.sh

```diff
@@ -420,8 +421,12 @@ function in_container_hostapp_update {
     fi
 
     while true ; do
-        ${DOCKER_CMD} pull "${update_package}" && break || log WARN "Couldn't pull docker image, was try #${retrycount}..."
-        retrycount=$[$retrycount+1]
+        if ${DOCKER_CMD} pull "${update_package}"; then
```

Comment:
these changes are to appease shellcheck

Reply:
:+1: I'm a fan of the more explicit notation too.